inception-concept-linking/src/main/resources/META-INF/asciidoc/user-guide/annotation_concept-linking.adoc

```diff
@@ -0,0 +1,30 @@
+[[sect_annotation_concept-linking]]
+=== Concept Linking
+==== Interactive Concept Linking
+When the user wants to link a mention, he marks it and starts typing the name of the concept.
+This triggers a pass through the entity linking pipeline and retrieves a ranked list of candidates,
+which is displayed to the user in the form of a drop-down menu.
+In order to enable a user to select the correct concept, it is crucial for him to receive textual
```

Comment:
I would just write that in order to know what entity he selected, a dropdown is shown.